sonar-db/src/main/java/org/sonar/db/user/GroupMembershipDao.java

```diff
@@ -74,34 +73,32 @@ public int countMembers(SqlSession session, UserMembershipQuery query) {
     return mapper(session).countMembers(params);
   }
 
-  public Map<String, Integer> countUsersByGroups(final DbSession session, Collection<Long> groupIds) {
-    final Map<String, Integer> result = Maps.newHashMap();
-    DatabaseUtils.executeLargeInputs(groupIds, new Function<List<Long>, List<GroupUserCount>>() {
-      @Override
-      public List<GroupUserCount> apply(@Nonnull List<Long> input) {
-        List<GroupUserCount> userCounts = mapper(session).countUsersByGroup(input);
-        for (GroupUserCount count : userCounts) {
-          result.put(count.groupName(), count.userCount());
-        }
-        return userCounts;
-      }
-    });
+  public Map<String, Integer> countUsersByGroups(DbSession session, Collection<Long> groupIds) {
+    Map<String, Integer> result = Maps.newHashMap();
+    executeLargeInputs(
+        groupIds,
+        input -> {
+          List<GroupUserCount> userCounts = mapper(session).countUsersByGroup(input);
+          for (GroupUserCount count : userCounts) {
+            result.put(count.groupName(), count.userCount());
+          }
+          return userCounts;
+        });
 
     return result;
   }
 
-  public Multimap<String, String> selectGroupsByLogins(final DbSession session, Collection<String> logins) {
-    final Multimap<String, String> result = ArrayListMultimap.create();
-    DatabaseUtils.executeLargeInputs(logins, new Function<List<String>, List<LoginGroup>>() {
-      @Override
-      public List<LoginGroup> apply(@Nonnull List<String> input) {
-        List<LoginGroup> groupMemberships = mapper(session).selectGroupsByLogins(input);
-        for (LoginGroup membership : groupMemberships) {
-          result.put(membership.login(), membership.groupName());
-        }
-        return groupMemberships;
-      }
-    });
+  public Multimap<String, String> selectGroupsByLogins(DbSession session, Collection<String> logins) {
+    Multimap<String, String> result = ArrayListMultimap.create();
+    executeLargeInputs(
+        logins,
+        input -> {
+          List<LoginGroup> groupMemberships = mapper(session).selectGroupsByLogins(input);
+          for (LoginGroup membership : groupMemberships) {
+            result.put(membership.login(), membership.groupName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)